src/main/java/mail/MailSender.java

```diff
@@ -0,0 +1,46 @@
+package mail;
+
+import javax.mail.*;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.util.Properties;
+
+public class MailSender {
+    private final GeneratorPassword randomPassword = new GeneratorPassword();
+
+    public void sendToEmail(String email, String password) {
+        final String from = "lovejudo96@mail.ru";
```

Comment:
1. why only this is final?
2. You create this string every time on `sendToEmail` method call. Can we create it once? It will never change